src/UI/AddBook.js

```diff
@@ -22,11 +36,28 @@ const AddBook = () => {
       </button>
       {isOpen && (
         <div>
-          <form>
-            <input type="text" placeholder="Title"></input>
-            <input type="text" placeholder="Author"></input>
-            <input type="text" placeholder="Genre"></input>
-            <button type="submit">Add Book!</button>
+          <form onSubmit={handleSubmit}>
+            <input
+              name='title'
+              value={formData.title || ''}
```

Comment:
Curious why the default here (the `||`). I don't think these values would ever be anything other than a string, right? Whether empty or not. So I think these defaults are redundant.

Reply:
Fair point. This was definitely new syntax for me. This is a situation where I would have used model in vue if I am remembering correctly. Basically it’s a way of modeling form data to a database/json file **I think. It’s been a couple months since I’ve worked with vue**